packages/leancode_flutter_svg_adaptive_loader/pubspec.yaml

```diff
@@ -0,0 +1,27 @@
+name: leancode_flutter_svg_adaptive_loader
+version: 1.0.0
+description: Flutter SVG adaptive loader for binary or xml-based formats
+homepage: https://github.com/leancodepl/flutter_corelibrary/tree/master/packages/leancode_flutter_svg_adaptive_loader
+repository: https://github.com/leancodepl/flutter_corelibrary
+
+environment:
+  sdk: ">=3.0.0 <4.0.0"
+  flutter: ">=3.10.0"
+
+dependencies:
+  flutter:
+    sdk: flutter
+  flutter_svg: ^2.0.7
+  vector_graphics: ^1.1.7
+  vector_graphics_compiler: ^1.1.7
+
+dev_dependencies:
+  flutter_test:
+    sdk: flutter
+  leancode_lint: ^4.0.0
+  meta: ^1.9.1
+
+flutter:
+  assets:
+    - assets/foo.svg
+    - assets/foo_compiled.svg
```

Comment:
Will that include these graphics for all users of this package?

Reply:
From my brief research:
1. Yes, including anything in package's `pubspec.yaml `will promote it also to users of the package.
2. There is no way of reading the asset file without having it defined in `pubspec.yaml`.
3. In Flutter framework tests they hack it by creating in test's `setUp` a new `pubspec.yaml`, writing by a buffer all the assets there and then testing what needs to be tested. https://github.com/flutter/flutter/blob/9cc4f94397f1f4b4d49ff38a02e5a0708dc9d68e/packages/flutter_tools/test/general.shard/asset_bundle_package_test.dart#L159

Reply:
From my brief research:
1. Yes, including anything in package's `pubspec.yaml `will promote it also to users of the package.
2. There is no way of reading the asset file without having it defined in `pubspec.yaml`.
3. In Flutter framework tests they hack it by creating in test's `setUp`

Reply:
I'd leave it attached since it is barely 2 KB

Reply:
I think attaching any asset regardless of its size is unwanted. Mati's suggestion of having a separate package sounds good though?

Reply:
Tackled in this PR https://github.com/leancodepl/flutter_corelibrary/pull/141